src/EventListener/ContainerScopeListener.php

```diff
@@ -40,9 +41,9 @@ public function __construct(ContainerInterface $container)
     /**
      * Enters the container scope when a route has been found.
      *
-     * @param FilterControllerEvent $event
+     * @param GetResponseEvent $event
      */
-    public function onKernelController(FilterControllerEvent $event)
+    public function onKernelRequest(GetResponseEvent $event)
```

Comment:
Because otherwise the container scope would not be set in the `InitializeSystemListener`